Makefile

```diff
@@ -1,8 +1,11 @@
+PROJECT_NAME:=mlxbf-bootctl
 SBINDIR = /sbin
 
 # Default target.
 all:
 
+include package.mk
```

Comment:
package.mk uses rpm commands.
There seems to be a problem that what if the build system doesn't support RPM?

Reply:
Tested on a debian system, it looks like the those specific calls will fail with a little error message, but the Makefile will move on. For completeness, I can check for rpm support and disable the RPM parts if rpm tools are not present on the system.